R/missingness.R

```diff
@@ -134,3 +135,41 @@ countMissingData <- function(x, userNAs = NULL) {
   .Deprecated("missingness", "healthcareai")
   missingness(x)
 }
+
+
+#' Replaces missing data with NA
+#'
+#' This function replaces any given value in a dataframe with NA.
+#'
+#' @param d A dataframe or matrix
+#' @param to_replace The value that will be replaced.
+#' @return A dataframe with the missing data replaced.
+#'
+#' @export
+#' @example
+#' # Replace "NULL" in dat
```

Comment:
I think you should change the example to have columns that make a little more sense. I'd use:
```
tibble(gender = c("male", "male", "female", "male", "missing"),
        age = c(64, 52, 75, "missing", 70),
        weight = c(139, 0, 193, 158, 273))
```
And then show how to remove these values.